lib/avrora/storage/registry.ex

```diff
@@ -93,10 +93,26 @@ defmodule Avrora.Storage.Registry do
   @spec configured?() :: true | false
   def configured?, do: !is_nil(registry_url())
 
+  defp csr_references_lookup_function(response) do
+    case Map.fetch(response, "references") do
+      {:ok, csr_references} ->
+        references = Enum.reduce(csr_references, %{}, fn(cr, map) ->
+          {:ok, schema_map} = get(cr["subject"])
```

Comment:
This will throw a matching error, but what if we can use `throw` to take control over it and return `{:error, ...}` instead. Also would be nice to cover it with a test of something like "reference can't be obtained"